app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<header class="navbar">
+  <h2>Sign up</h2>
+</header>
+
+<div class= 'form sign-up'>
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+      <%= f.label :name %><br />
+      <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+    </div>
+    
+    <div class="field">
+      <%= f.label :email %><br />
+      <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+    </div>
+
+    <div class="field password-field">
+      <%= f.label :password %>
+      <% if @minimum_password_length %>
+      <em>(<%= @minimum_password_length %> characters minimum)</em>
+      <% end %>
+      <%= f.password_field :password, autocomplete: "new-password" %>
+    </div>
+
+    <div class="field">
+      <%= f.label :password_confirmation %><br />
+      <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+    </div>
+
+    <div class="actions">
+      <%= f.submit "Sign up", class: "save-btn"%>
+    </div>
+  <% end %>
+
+  <%= render "devise/shared/links" %>
+</div>
```

Comment:
- Please make sure to stick to the original design as much as possible, which includes:
   - [ ] Using the same typography and font style as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Typography%3A%20font%20face%2C%20size%20and%20weight.) requirement.
   - [ ] Changing the positioning of the "Submit" button to be in the nav which is then named "Next".
   - [ ] Editing the style of the inputs to look more like the original template.
   - [ ] Adding a "left arrow" icon to the left part of the nav which acts as a back button to match the given template.
   - [ ] Using the same color in the nav and the text color as well, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=guidelines%2C%20including%3A-,Colors.,-Typography%3A%20font%20face) requirement.
   - [ ] Changing the background of the page to match the one in the design.
   
(This goes for all other forms as well even if they don't have a direct style guideline, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=NOTE%3A%20In%20these%20design%20guidelines%20there%20are%20several%20UIs%20that%20you%20won%27t%20need%20for%20this%20exercise%3B%20also%2C%20some%20pages%20are%20not%20given%20a%20design%20and%20you%20will%20create%20them%20following%20the%20design%20guidelines%20of%20the%20other%20pages.) requirement)
| Your design | Original design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/235155462-cec028b8-1c4c-4327-8af3-9225b0f1817a.png) | ![image](https://user-images.githubusercontent.com/62290545/235155597-22d2f993-9767-448e-b94e-afe12ad50d64.png) |